server/src/controllers/authController.ts

```diff
@@ -0,0 +1,67 @@
+import { UserModel } from "../dataBase/descriptionDB";
```

Comment:
Let's think about aliases and the order of the imports

https://christianlydemann.com/simpler-typescript-paths-with-path-aliases/#:~:text=Typescript%20aliases%20allow%20you%20to,%2Fapp%2Fconfig%2Fconfig.